.env.example

```diff
@@ -46,3 +46,6 @@ EMAIL_HOST_PASSWORD=
 EMAIL_HOST_USER=
 EMAIL_HOST_PORT=
 EMAIL_FROM=
+PORT=_integer_
```

Comment:
Does the `_integer_` etc provide info to some IDE?  If so, this is cool.  However, if it's just a textual hint, I'd rather they be blank in this file, so when people copy it over, if they don't modify the line at all, then it stays an empty value (and e.g. tests if the value exist will do the right thing)